src/styles/layout.css

```
@@ -12,6 +12,11 @@ body {
     display: flex;
     flex-direction: column;
     font-family: system-ui, sans-serif;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.7.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)